src/Tasks/Resources/Strings.resx

```diff
@@ -1967,6 +1967,10 @@
     <value>MSB3473: Path for "{0}" is invalid. {1}</value>
     <comment>{StrBegin="MSB3473: "}</comment>
   </data>
+  <data name="SGen.TaskNotSupported">
+    <value>MSB3474: The task "{0}" is not supported on the .NET Core version of MSBuild. Please instead use the Microsoft XML Serializer Generator. See https://go.microsoft.com/fwlink/?linkid=858594 for further details.</value>
```

Comment:
Tweaks--@kathleendollard, do you have any input on this error?

```suggestion
    <value>MSB3474: The task "{0}" is not supported on the .NET Core version of MSBuild. Use the Microsoft XML Serializer Generator package instead. See https://go.microsoft.com/fwlink/?linkid=858594 for more information.</value>
```

Reply:
Synced up with @timheuer on this in Kathleen's absence. Tim is fine with the error but points out that the doc article it links to needs some love. Following up in this issue: https://github.com/dotnet/docs/issues/14862

Reply:
I've tweaked the error as you suggested. Thank you!